docs/cpp/config-mingw.md

```diff
@@ -25,14 +25,17 @@ To successfully complete this tutorial, you must do the following steps:
 
     ![C/C++ extension](images/cpp/cpp-extension.png)
 
-1. You will install Mingw-w64 via the SourceForge website. Click [Mingw-w64](https://sourceforge.net/projects/mingw-w64/files/Toolchains%20targetting%20Win32/Personal%20Builds/mingw-builds/installer/mingw-w64-install.exe/download) to begin downloading the compressed archive file. Extract the tools from the compressed file to a folder that has no spaces in its path. In this tutorial, we assume it is installed under `C:\mingw-w64`.
+1. Install Mingw-w64 via the SourceForge website. Click [Mingw-w64](https://sourceforge.net/projects/mingw-w64/files/Toolchains%20targetting%20Win32/Personal%20Builds/mingw-builds/installer/mingw-w64-install.exe/download) to download the Windows Mingw-w64 installer.
+    1. Run the installer.
+    1. For **Architecture** select **x86_64** and then select **Next**.
+    1. **Next** again to use the default installation folder and install Mingw.
 
-1. Add the path to your Mingw-w64 `bin` folder to the Windows PATH environment variable.
+1. Add the path to your Mingw-w64 `bin` folder  to the Windows `PATH` environment variable by using the following steps:
    1. In the Windows search bar, type 'settings' to open your Windows Settings.
-   1. Search for **Edit environment variables for your account**.
+   1. Search for **Edit environment variables** for your account**.
```

Comment:
The UI text is "Edit environment variables for your account". You would also have trailing **.
![image](https://user-images.githubusercontent.com/12818376/88069745-bb500200-cb26-11ea-93db-5659fb80d9d2.png)